app/src/main/java/com/github/geohunt/app/data/repository/ChallengeRepository.kt

```diff
@@ -173,4 +169,14 @@ class ChallengeRepository(
 
         return challengeEntry.asExternalModel(challengeId)
     }
+
+    /**
+     * Returns all the claims done by a specific user id
+     *
+     * @param uid The user id
+     */
+    override fun getClaimsFromUser(uid: String): List<Claim> {
+        // TODO
+        return listOf()
+    }
```

Comment:
Deprecated in my PR, this should really go in the `ClaimRepository`

Reply:
Yeah on that note, I'll fix it myself when resolving conflict for my PR

Reply:
if I don't forget :eyes: @BoyeGuillaume

Reply:
For sure then!